.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
       - image: circleci/golang:1.13
     environment:
       - TEST_RESULTS: /tmp/test-results # path to where test results are saved
-      - CONSUL_VERSION: 1.6.3 # Consul's OSS version to use in tests
```

Comment:
This is required right now because the code in this branch uses the node list endpoint that's only in 1.7. With #205 this could be put back to 1.6.3.